### ISSUE TITLE
Missing metadata.xml when recording screenshots in an Android library module

### ISSUE THREAD

**Issue by AlexKrupa** [Karumi/Shot#112]
### Expected behaviour
Recording screenshots works in a `com.android.library` module.

### Actual behaviour
Build fails with:
```
Execution failed for task ':samplelibrary:debugDownloadScreenshots'.
> java.io.FileNotFoundException: Source '<trimmed>/Shot/shot-consumer/samplelibrary/screenshots/debug/screenshots-default/metadata.xml' does not exist
```

Similar task works just fine in `com.android.application` modules (e.g. `app` and `app2` in `shot-consumer` project).

### Steps to reproduce
Tissue also occurs in the Shot's sample project.

1. Open `shot-consumer` project.
2. Execute this shell command:
```
./gradlew :samplelibrary:executeScreenshotTests
```

There's no difference between emulator and physical devices.

### Version of the library
4.2.0, 4.2.1-SNAPSHOT

### Additional info
I found this issue https://github.com/Karumi/Shot/issues/55 reporting a similar build failure, but the solution doesn't apply here.

**Comment by pedrovgs** [comment]
Hi @AlexKrupa I've just double-checked the current implementation of the library and executed the tests for the ``samplelibrary`` module and everything works fine. Actually, the CI workflow executes those tests too and the build is passing in master and the latest pull requests. Can you please check if the emulator you used to run the tests has an SD card associated? Could you also check if there is something in the device logcat after executing the test suite? The issue could be related to the tests execution and not the gradle plugin :thinking:

**Comment by AlexKrupa** [comment]
Thanks for your reply, it in the right direction. I finally got it to work.

`create_emulator.sh` script executes `$ANDROID_HOME/tools/mksdcard`. Since that one is a 32-bit executable I couldn't run it on MacOS Catalina. I wasn't aware of the 64-bit `mksdcard` in `$ANDROID_HOME/emulator/mksdcard`. To work around that I ran the tests on an IDE-created emulator, which was good enough for preparing a PR (unrelated).

Everything's fine after pointing to 64-bit `mksdcard` and letting the script create the emulator. I'm still not sure what the issue was exactly with other emulators/devices.

---

Taking the above into consideration, should the script in the repo be updated to point to 64-bit `mksdcard` as well?

Other than that, I'm closing the issue as resolved.
